### DISCUSSION TITLE
Groups?

### DISCUSSION THREAD

**Discussion by derekmahar** [balzack/databag#68] [Q&A] [answered]
Does Databag support the concept of "groups" or "rooms" where multiple participants may exchange messages visible only to them and not to non-members?  Do rooms support message topics similar to those that exist between two users?  (Conceptually, one might think of the private communication between two users as just a special case of a room that contains just two participants.)

**Comment by balzack** [comment]
Current Databag does supports the concept of "groups" but not with anonymous participants. The one hosting the topics manages the member list which are selected from their connected contact list. For your use case, is anonymous participants an important feature?

Thanks for your interest in this project!

**Reply by derekmahar** [reply]
Sorry if my question was unclear, but in the scenario that I'm considering, the participants in the group need not be anonymous.  However, the messages in the group should be visible only to members of the group and not to non-members.  In effect, the messages in the group are private to those members.

**Reply by balzack** [reply, marked as answer]
Ah, I see. Yes, group conversations are supported and they are private to the members of the 'topic'; non-members will not have access to the topic or messages. Please let me know if you have any issues or if you see ways for it to be improved.
